JavaScript homeworks/homework-2.js

```diff
@@ -0,0 +1,24 @@
+let weightSarah = 65;
+let weightBob = 93;
+
+let heightSarah = 1.75;
+let heightBob = 1.89;
+
+let sarahBMI = weightSarah / heightSarah ** 2
+let bobBMI = weightBob / heightBob ** 2
+
+console.log (sarahBMI)
+console.log (bobBMI)
+
```

Comment:
Technically all is correct. 
But in task was required to use operator AND for Sarah and OR for Bob. 

SO your logic for Sarah might be  
`if (sarahBMI < 18.5) {
console.log ("Sarah is underweight")
} else {(sarahBMI >= 18.5 && sarahBMI < 25) 
    console.log ("Sarah has a normal weight")
}`

Reply:
oh, thank you, I fixed it!